### ISSUE TITLE
Accessibility: Intro videos should provide open captions (subtitles)

### ISSUE THREAD

**Issue by brownbonnie** [microsoft/vscode-docs#4892]
Documentation does not meet accessibility requirements for users with additional access needs. The only format provided is video, where a transcript should also be provided:

E.g. https://code.visualstudio.com/docs/introvideos/codeediting

The video also does not play on Linux. Error code "We're sorry, an error has occurred when playing video (video failed to download)"

**Comment by gregvanl** [comment]
Hi @brownbonnie I've upgraded my Linux machine to 20.04.3 and made sure I'm on the latest Firefox for Ubuntu 93.0 but I haven't been able to reproduce the video not running.
I tried two popular ad blockers, AdBlock and AdBlock Plus, but they don't seem to cause an issue.
Can you specify the ad blocker you are using? Thanks

**Comment by gregvanl** [comment]
Looking at recent comments, earlier in the month I see people saying the video won't play but not in the last two weeks.
Could you try to watch the video again as I'm wondering if there was a server outage.
Thank you for your help

**Comment by gregvanl** [comment]
I wonder if you are missing a video codec https://linuxhint.com/install_multimedia_codecs_ubuntu/
I see people recommending the `ubuntu-restricted-extras` to install codecs

**Comment by brownbonnie** [comment]
Excellent thank you, yes I was missing the codecs. Following the guide you linked worked great.

**Comment by brownbonnie** [comment]
Please could this issue be kept open for the accessibility side of things, e.g. transcripts. Many thanks :)

**Comment by gregvanl** [comment]
Glad you are unblocked. 
I'll add a note on the /docs/getstarted/introvideos page for Linux users in case other people hit this issue.
Yes, we should keep this issue open. I've updated the title to call out the need for video transcripts.

**Comment by gregvanl** [comment]
Add mention of codecs installation with https://github.com/microsoft/vscode-docs/commit/fab0a96a9f7a3d38b0929552202ef97cd8604ca3

**Comment by brownbonnie** [comment]
Thank you :)

**Comment by hughlilly** [comment]
Hello. Apologies for leaving this comment here but I think it's related enough that I don't need to start a new issue. There is a word missing from the captions for this video:

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/12046008/144346147-120e8b3e-caab-45d2-9d2b-3fb14115ba0c.png">

It should say "have richer experiences". I wonder if your video plans include open-sourcing the captions, from which the transcripts would presumably be drawn? This would allow for corrections such as this.

Thanks
Hugh

**Comment by brownbonnie** [comment]
@hughlilly Agreed - open-sourcing the captions could be a really good idea, assuming any changes would be peer-reviewed by the repo maintainers as per the usual process of course.

**Comment by chrisdias** [comment]
ToDo:

- caption every video with SRT generated by video indexer
- publish the SRT files to an public repository
- add a link to the SRT from the doc/video page

**Comment by ilyachalov** [comment]
@gregvanl @brownbonnie The transcript can be obtained as follows:

![transcript](https://user-images.githubusercontent.com/48008056/165396132-73f11dcc-9a13-481d-b7a8-3535ad3091bf.png)

I think the issue can now be renamed to 'Accessibility: Intro videos should provide captions (subtitles)' or something similar.

**Comment by ilyachalov** [comment]
There are errors in the subtitles, for example, in the introvideo '[Productivity Tips](https://code.visualstudio.com/docs/introvideos/productivity)':

**time stamp 01:08**
_caption:_ 'You can do the same thing with the **add (+)** sign to go to a symbol in a file.'
_correctly:_ 'You can do the same thing with the **at (@)** sign to go to a symbol in a file.'

**time stamp 02:36**
_caption:_ 'You can also set Editor Format On **Pace** to True to format your text whenever you paste new text into your editor.'
_correctly:_ 'You can also set 'Editor: Format On **Paste**' to 'True' to format your text whenever you paste new text into your editor.'

**time stamp 03:27**
_caption:_ 'You can trigger IntelliSense in any editor window by typing **carrot** space (^ ) or by typing a trigger character such as the dot (.) character in JavaScript.'
_correctly:_ 'You can trigger IntelliSense in any editor window by typing **caret** space (^ ) or by typing a trigger character such as the dot (.) character in JavaScript.'

In addition, I would like to add subtitles in Russian.

**Comment by brownbonnie** [comment]
Many thanks @ilyachalov I have renamed the issue

**Comment by ilyachalov** [comment]
@chrisdias I created subtitles for introvideo «[Productivity Tips](https://code.visualstudio.com/docs/introvideos/productivity)» using «[Azure Video Indexer](https://www.videoindexer.ai/)»:

[productivity-vi.srt](https://github.com/ilyachalov/ru-vscode-introvideo-subtitles/blob/main/productivity-vi.srt)

These subtitles are very different from the subtitles in the original introvideo.

I changed them manually:

[productivity-en.srt](https://github.com/ilyachalov/ru-vscode-introvideo-subtitles/blob/main/productivity-en.srt)

Translation into Russian:

[productivity-ru.srt](https://github.com/ilyachalov/ru-vscode-introvideo-subtitles/blob/main/productivity-ru.srt)

These subtitles can be used, but the timestamps are not set perfectly. Is it possible to get the original subtitles with the correct timestamps? Then you won't need video indexer!

**Comment by ilyachalov** [comment]
I created subtitles for introvideo '[Getting started](https://code.visualstudio.com/docs/introvideos/basics)' which is embedded from 'Youtube':

in English ( @theReynald )
[basics-en.srt](https://github.com/ilyachalov/ru-vscode-introvideo-subtitles/blob/main/basics-en.srt)

in Russian (my translation)
[basics-ru.srt](https://github.com/ilyachalov/ru-vscode-introvideo-subtitles/blob/main/basics-ru.srt)

**Comment by jaguarfi** [comment]
@gregvanl if I want to add videos to the documentation which screen recorder should I use? or what are the guidelines for recording such videos?

**Comment by stech11845** [comment]
I think this can get into the hands of a video professional.

**Comment by gregvanl** [comment]
Closing old issue. Closed captions are available for the videos hosted on YouTube and www.microsoft.com.
No plans currently to open source the captions or provide localized captions.
